### PR TITLE
Samples

### PR DESCRIPTION
**Pull request by brunoborges**
New samples

### CODE REVIEW

**Comment by maxandersen** [comment]
hey! that’s awesome ;) getting late here - i'll have to look in details tomorrow ;)

**Comment by maxandersen** [comment]
note., the failure is just due to me enabling windows tests and there are failures atm. your PR passed what it needed to.

File: examples/webapps/undertow/index.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+/**
```
Comment (maxandersen) [inline examples/webapps/undertow/index.js]: are these nashorn related files needed/relevant for jbang usecase ?

File: examples/webapps/spark/sparkjava.java [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+        get("/hello", (req, res) -> {
+            return "Hello Spark!";
+        });
+
+        get("/json", "application/json", (req, res) -> {
+            var date = java.util.Calendar.getInstance().getTime();
+            var obj = {"name":"Nasven.js", "date":"${date}"};
+            return obj;
+}, function(model) JSON.stringify(model));
```
Comment (maxandersen) [inline examples/webapps/spark/sparkjava.java]: this gives a compile error ...?

**Comment by brunoborges** [comment]
Urgh, ignore this PR for now. I need to redo the code. Had issues with git and ended up rolling back too many commits.

**Pull request closed without merging**